lib/Upcoming_Ride.dart

```diff
@@ -1,277 +1,85 @@
-                        children: <Widget>[
-                          Text('Uris Hall', style: infoStyle)
-                        ],
-                      )
-                  ),
-                  Container(
-                      margin: EdgeInsets.symmetric(horizontal: 15.0),
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.start,
-                        children: <Widget>[
-                          Text('To', style: directionStyle)
-                        ],
-                      )
-                  ),
-                  Container(
-                      margin: EdgeInsets.symmetric(horizontal: 15.0),
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.start,
-                        children: <Widget>[
-                          Text('Cascadilla Hall', style: infoStyle)
-                        ],
-                      )
-                  ),
-                  Container(
-                      margin: EdgeInsets.symmetric(horizontal: 15.0),
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.start,
-                        children: <Widget>[
-                          Text('Ride Confirmed', style: confirmationStyle)
-                        ],
-                      )
-                  ),
-                  Container(
-                    margin: EdgeInsets.symmetric(horizontal: 15.0),
-                    child: Row(
-                      children: <Widget>[
-                        Padding(
-                          padding: const EdgeInsets.only(top: 8.0),
-                          child: Icon(Icons.person_pin, size: 38.0),
-                        ),
-                        SizedBox(width: 5.0),
-                        Column(
-                          crossAxisAlignment: CrossAxisAlignment.start,
-                          children: <Widget>[
-                            Text('Davea Butler', style: infoStyle),
-                            Row(
-                              children: <Widget>[
-                                Icon(Icons.phone, size: 13.0,),
-                                SizedBox(width: 5.0),
-                                GestureDetector(
-                                  onTap: () => UrlLauncher.launch("tel://13232315234"),
-                                  child: Text('+1 323-231-5234', style: TextStyle(color: Colors.grey[600])),
-                                )
-                              ],
-                            )
-                          ],
-                        )
-                      ],
-                    ),
-                  )
-                ],
-              ),
+          UpcomingRideCard(
+            rideStatus: Text('Ride Confirmed', style: confirmationStyle),
+            timeDateWidget: RichText(
+              text: TextSpan(text: 'OCT' + ' ', style: monthStyle, children: [
```

Comment:
For your next PR, instead of hardcoding these dates and times, try to connect the cards to the backend.